plugins/inputs/ipset/ipset_entries_test.go

```diff
@@ -0,0 +1,122 @@
+package ipset
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestIpsetEntries(t *testing.T) {
+	acc := new(testutil.Accumulator)
+
+	i := ipsetEntries{}
+	if i.addLine("create mylist hash:net family inet hashsize 16384 maxelem 131072 timeout 300 bucketsize 12 initval 0x4effa9ad", acc) != nil {
+		t.Error("addLine returned an error")
+	}
+
+	if i.addLine("add mylist 89.101.238.143 timeout 161558", acc) != nil {
+		t.Error("addLine returned an error")
+	}
+
+	if i.addLine("add mylist 122.224.15.166 timeout 186758", acc) != nil {
+		t.Error("addLine returned an error")
+	}
+
+	if i.addLine("add mylist 47.128.40.145 timeout 431559", acc) != nil {
+		t.Error("addLine returned an error")
+	}
+
+	i.commit(acc)
```

Comment:
Could we please use the power of the `require` package here!? Same for the tests below...
```suggestion
	lines := []string{
		"create mylist hash:net family inet hashsize 16384 maxelem 131072 timeout 300 bucketsize 12 initval 0x4effa9ad",
		"add mylist 89.101.238.143 timeout 161558",
		"add mylist 122.224.15.166 timeout 186758",
		"add mylist 47.128.40.145 timeout 431559",
	}

	entries := ipsetEntries{}
	
	var acc testutil.Accumulator
	for _, line := range lines {
		require.NoError(t, i.addLine(line, &acc))
	}
	i.commit(acc)
```

Reply:
sure we can do that!